### DISCUSSION TITLE
Custom GM event before black temple opening

### DISCUSSION THREAD

**Discussion by Kirishiki** [chromiecraft/chromiecraft#7223] [Suggestions]
before black temple opens , there could be a cool event from GM´s where demons try to take over horde or alliance main citys led by illidan, like first to stormwind, with waves of demons rushing in towards king. if players manage to stop them they all get some cool rewards. ofc after would be orgrimmar and same thing there . limiting pvp off tho, so opposite faction dont come ruin it in any way.

**Comment by joesill** [comment]
They should spawn dozens of archimondes that spam doomfire all over the world.
